packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/VirtualAssistant/VirtualAssistant.md

```diff
@@ -21,6 +21,8 @@ import LoadingMessage from '@patternfly/virtual-assistant/dist/dynamic/LoadingMe
 import { GrinIcon } from '@patternfly/react-icons';
 import { AngleDownIcon } from '@patternfly/react-icons';
 import ConversationAlert from '@patternfly/virtual-assistant/dist/esm/ConversationAlert'
```

Comment:
```suggestion
import ConversationAlert from '@patternfly/virtual-assistant/dist/dynamic/ConversationAlert'
```
I know it's not your change, but can you switch this one to dynamic? Thank you

Reply:
Fixed, also fixed for `SystemMessageEntry`